scripts/python_installer

```diff
@@ -24,7 +24,9 @@ BASH_PROFILE="$HOME/.bash_profile"
 ZSHENV="$HOME/.zshrc"
 COMMAND_LINE_ARGS=$*
 PYTHON_VERSION=${PYTHON_VERSION:="3.7.2"}
+SURPRESS_PATH_EXPORT_MESSAGE=${SURPRESS_PATH_EXPORT_MESSAGE:="false"}
```

Comment:
We should be able to booleans, `true` and `false`, rather than the strings, "true" and "false".